guides/gatherers.md

```diff
@@ -381,3 +382,22 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/systemd_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/systemd_test.go)
+
+### verify_password
+
+**Argument required**: yes.
+
+This gatherer uses `getent` to determine whether a password from a certain user needs to be changed or not. It returns `true` if the password should be changed or `false` if not.
+
+Specification examples:
+```yaml
+facts:
+  - name: hacluster
+    gatherer: verify_password
```

Comment:
Given that the responsibility of this gatherer is to tell whether a well known password is detected for a user or not (and it won't change much afaiu), I'd see fit also a more specific name like `verify_known_password` or similar, rather than`verify_password`, which I find a bit distant from what is actually happening.

How would that work?

Reply:
I wouldn't change this. Even if we don't plan to change in it the neart future, this gatherer could be easily improved to add additional verifications for the password (i.e. "is this password based on dictionary words?" or "is this password too short?", ...). It would be a pain in that case to update all checks to match this.

wdyt?

Reply:
Naming is always a pandora box :smile: no worries, keep it as you wish.